MFiles.VAF.Extensions.Tests/ExtensionMethods/ObjVerEx/GetPropertyAsValueListItems.cs

```diff
@@ -0,0 +1,57 @@
+﻿using MFiles.VAF.Configuration;
+
+using MFilesAPI;
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using Moq;
+
+using System;
+
+namespace MFiles.VAF.Extensions.Tests.ExtensionMethods.ObjVerEx
+{
+    [TestClass]
+    public class GetPropertyAsValueListItems
+        : TestBaseWithVaultMock
+    {
+
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void ThrowsIfNullObjVerEx()
+        {
+            ((Common.ObjVerEx) null).GetPropertyAsValueListItems((int) MFBuiltInPropertyDef.MFBuiltInPropertyDefWorkflow);
+        }
+
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentOutOfRangeException))]
+        public void ThrowsIfPropDefIdNegative()
+        {
+            ((Common.ObjVerEx) null).GetPropertyAsValueListItems((int) -1);
```

Comment:
This test does not pass.  The exception throws an ArgumentNullException - did you mean to use a valid ObjVerEx instance?